test/jquery.i18n.test.js

```diff
@@ -406,7 +406,7 @@
 
 	function grammarTest( langCode, test ) {
 		QUnit.test( 'Grammar test for language ' + langCode, function ( assert ) {
```

Comment:
The other tests in this file are still using `QUnit( '..', Number, function)` which isn't supported in QUnit 2.x. Might wanna fix those as well, or upgrade in a separate commit from the switch to npm.